### PR TITLE
added OrderDao and Service

### PR DESCRIPTION
**Pull request by akutil**

### CODE REVIEW

File: src/main/java/com/gmail/kutilandrej/cinema/service/impl/OrderServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import com.gmail.kutilandrej.cinema.service.OrderService;
+import com.gmail.kutilandrej.cinema.service.ShoppingCartService;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+
+    @Inject
+    OrderDao orderDao;
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/gmail/kutilandrej/cinema/service/impl/OrderServiceImpl.java]: access modifier

File: src/main/java/com/gmail/kutilandrej/cinema/service/impl/OrderServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    @Inject
+    ShoppingCartService shoppingCartService;
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/gmail/kutilandrej/cinema/service/impl/OrderServiceImpl.java]: same

File: src/main/java/com/gmail/kutilandrej/cinema/dao/impl/OrderDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            }
+        }
+    }
+
+    @Override
+    public List<Order> getOrderHistory(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            String hql = "FROM Order O JOIN FETCH O.tickets AS T JOIN FETCH T.movieSession "
+                    + "AS MS JOIN FETCH MS.movie JOIN FETCH MS.cinemaHall "
+                    + "JOIN FETCH T.user WHERE O.user = :user";
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/gmail/kutilandrej/cinema/dao/impl/OrderDaoImpl.java]: Can we do that with fetching only o.tickets?

Comment (akutil) [reply src/main/java/com/gmail/kutilandrej/cinema/dao/impl/OrderDaoImpl.java]: okay

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**